### PR TITLE
VidCloud9 Update

### PR DESCRIPTION
**Pull request by Gurrman375**
Fixed all the errors left in the previous build.
Proof:
image
image
image
image
Settings Proof:
image
image
image

### CODE REVIEW

File: websites/V/VidCloud9/presence.ts [hunk 1]
```diff
@@ -23,114 +31,120 @@ presence.on("iFrameData", (data) => {
   }
 });
 
-function getTimestamps(videoTime, videoDuration): any {
-  var startTime = Date.now();
-  var endTime = Math.floor(startTime / 1000) - videoTime + videoDuration;
+function getTimestamps(videoTime: number, videoDuration: number): Array<number> {
+  const startTime = Date.now(), endTime = Math.floor(Math.floor(startTime / 1000) - videoTime + videoDuration);
   return [Math.floor(startTime / 1000), endTime];
 }
```
Comment (ririxidev) [inline websites/V/VidCloud9/presence.ts:37]: presence.getTimestamps is in presence class already

File: websites/V/VidCloud9/presence.ts [hunk 2]
```diff
@@ -23,114 +31,120 @@ presence.on("iFrameData", (data) => {
...
-  const info = await presence.getSetting("sSI");
-  const elapsed = await presence.getSetting("sTE");
-  const videoTime = await presence.getSetting("sVT");
+  const info = await presence.getSetting("sSI"), elapsed = await presence.getSetting("sTE"), videoTime = await presence.getSetting("sVT");
 
   if (elapsed) {
-    var browsingStamp = Math.floor(Date.now() / 1000);
-    console.log("Elapsed is on");
+    browsingStamp = Math.floor(Date.now() / 1000);
+    presence.info("Elapsed is on");
```
Comment (ririxidev) [inline websites/V/VidCloud9/presence.ts:44]: presence.info is only available when you will enable them in metadata by ``readLogs`` to true

**Review by ririxidev — CHANGES_REQUESTED** [review]

File: websites/V/VidCloud9/presence.ts (hunk 2 above)
Comment (Gurrman375) [reply websites/V/VidCloud9/presence.ts:44]: If that's the case then `readLogs` is true by default.

Comment (ririxidev) [reply websites/V/VidCloud9/presence.ts:39]: a no wait lol ,you are right 🤦 . readlogs are only for getting logs bruh

Comment (ririxidev) [reply websites/V/VidCloud9/presence.ts:39]: Fixed guidelines btw about it lol

**Review by ririxidev — APPROVED** [review]
looks fine for me 👍 sorry for confusion

File: websites/V/VidCloud9/presence.ts (hunk 2 above)
Comment (Gurrman375) [reply websites/V/VidCloud9/presence.ts:39]: cool

File: websites/V/VidCloud9/dist/metadata.json [hunk 3]
```diff
@@ -18,6 +19,7 @@
   ],
   "category": "videos",
   "iframe": true,
+  "readLogs": false,
   "settings": [
```
Comment (Bas950) [inline websites/V/VidCloud9/dist/metadata.json:23]: no need to set to false since thats the default value

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/V/VidCloud9/dist/metadata.json (hunk 3 above)
Comment (Gurrman375) [reply websites/V/VidCloud9/dist/metadata.json:23]: okay then I'll change that

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**